scss/main.scss

```diff
@@ -7,5 +9,52 @@
 .container {
   display: flex;
   flex-direction: column;
-  padding: 71px 20px 0 20px;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  padding: 71px 15px 38px 15px;
+
+  & .header-title {
+    color: $primary-color;
```

Comment:
If you're repeating this property all over the stylesheet, you should consider using it in a common ancestor, selector, etc.